worker/config.go

```diff
@@ -199,6 +201,18 @@ func ConfigSchema() schematypes.Object {
 				Minimum: 0,
 				Maximum: 65535,
 			},
+			"networkInterface": schematypes.String{
+				MetaData: schematypes.MetaData{
+					Description: "Network device webhookserver should listen on, if not serverIp",
```

Comment:
nit: "Network device webhookserver should listen on. If not supplied, it binds to the interface from serverIp address."